content/docs/deployment/helm/modules/installation/observability/_index.md

```diff
@@ -547,49 +547,21 @@ In this case all storage system requests made by CSM for Observability will not
-   done
+   kubectl delete secret powermax-config -n [CSM_NAMESPACE]
    ```
 
-2. Delete the current `powermax-reverseproxy-config` configmap from the CSM namespace.
-   ```console
-
-   kubectl delete configmap powermax-reverseproxy-config -n [CSM_NAMESPACE] 
-   ```
-
-3. Copy the configmap `powermax-reverseproxy-config` from the CSI Driver for Dell PowerMax namespace to the CSM namespace.  
+2. Copy the Secret `powermax-config` from the CSI Driver for Dell PowerMax namespace to the CSM namespace.
    
    ```console
-
-   kubectl get configmap powermax-reverseproxy-config -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -
+   kubectl get secret powermax-config -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -
    ```
 
-   If the CSI driver configmap name is not the default `powermax-reverseproxy-config`, please use the following command to copy configmap:
+   If the CSI driver secret name is not the default `powermax-config`, please use the following command to copy the secret:
 
    ```console
 
-   kubectl get configmap [POWERMAX-REVERSEPROXY-CONFIG] -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/name: [POWERMAX-REVERSEPROXY-CONFIG]/name: powermax-reverseproxy-config/' | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -
+   kubectl get secret [POWERMAX-CONFIG] -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/name: [POWERMAX-CONFIG]/name: powermax-config/' | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | kubectl create -f -
    ```
-
-4. Copy the secrets in `powermax-reverseproxy-config` from the CSI Driver for Dell PowerMax namespace to the CSM namespace.  
-    ```console
-
-    for secret in $(kubectl get configmap powermax-reverseproxy-config -n [CSI_DRIVER_NAMESPACE] -o jsonpath="{.data.config\.yaml}" | grep arrayCredentialSecret | awk 'BEGIN{FS=":"}{print $2}' | uniq)
-    do
-       kubectl get secret $secret -n [CSI_DRIVER_NAMESPACE] -o yaml | sed "s/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/" | kubectl create -f -
-    done
-    ```
-
-    If the CSI driver configmap name is not the default `powermax-reverseproxy-config`, please use the following command to copy secrets:
-    ```console
-
-    for secret in $(kubectl get configmap [POWERMAX-REVERSEPROXY-CONFIG] -n [CSI_DRIVER_NAMESPACE] -o jsonpath="{.data.config\.yaml}" | grep arrayCredentialSecret | awk 'BEGIN{FS=":"}{print $2}' | uniq)
-    do
-       kubectl get secret $secret -n [CSI_DRIVER_NAMESPACE] -o yaml | sed "s/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/" | kubectl create -f -
-    done
-    ```
+   **Note:** The config map approach is being phased out. However, for backwards compatibility, you can still configure and use the Observability module with PowerMax driver using the config map.
```

Comment:
Can we say "config map approach will be deprecated in upcoming releases of CSM" instead of phased out

Reply:
Done. Made similar change here as well to maintain consistency across the documentation - https://github.com/dell/csm-docs/pull/1448/files#diff-aaf67fc132e1164de1a014c93869493e31f634992adf0430a4f0f4b2b424af66L709

@lukeatdell